codonPython/tests/suppression_test.py

```diff
@@ -1,16 +1,16 @@
-from codonPython.suppression import suppress_value
+from codonPython.suppression import central_suppresion_method
```

Comment:
@NatashaChetwynd The build is failing because `suppresion` is spelt as `suppression` in the module. Changing this should fix the build

Reply:
@georgeRobertson I've made this change to the Con-154_adjust_suppression_function branch